src/test/suite/camel.version.settings.test.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+import { ShellExecution, workspace, tasks, Task } from 'vscode';
+import { CamelJBangTaskProvider } from '../../task/CamelJBangTaskProvider';
+import { expect } from 'chai';
+import { getCamelTask, getTaskCommandLine } from './util';
+
+suite('Should run commands with Camel version specified in settings', () => {
+
+	const CAMEL_VERSION = '3.20.1';
+    const CAMEL_VERSION_SETTINGS_ID = 'camel.debugAdapter.CamelVersion';
+
+	let defaultCamelVersion = '';
+
+	suiteSetup(function () {
+		defaultCamelVersion = workspace.getConfiguration().get(CAMEL_VERSION_SETTINGS_ID) as string;
```

Comment:
I think this is more accurate to call it `initialCamelVersion`. In case, the test suite is launched with specific settings, the version will be different. I know we are not doing it currently but this might be interesting one day if we have enough maturity fo rthe project to be able to launch the whole test suite with various environments, various productized Camel version for instance.

Reply:
agree, changed